src/pages/CategoriesPage.js

```diff
@@ -4,7 +4,7 @@ function CategoriesPage() {
   return (
     <div>
       <div className="categories-wrapper row">
-        <button type="button">Check status</button>
+        <button type="button" onClick={() => console.log('Check Status is under construction')}>Check status</button>
```

Comment:
- [ ] Please as the last reviewer said check CategoriesStatus should dispatch the under-construction action creator as I can see you just console.log the "under-construction" word on the dev tools, please change it to dispatch the action to the store.
![image](https://user-images.githubusercontent.com/74744787/222797946-3c3e2ba6-d2ec-45f0-ac68-6dcffc4c7f15.png)

Reply:
Thanks I have Changed It